### PR TITLE
Create 2017-06-05-SteeringCommitteeRetreat.md

### PR DESCRIPTION
**Pull request by raynamharris**
Summarizes the in person meeting.  

Needs to first be approved by @ChristinaLK @k8hertweck @mkuzak @karinlag @weaverbel @smcclatchy

### CODE REVIEW

File: _posts/2017/06/2017-06-05-SteeringCommitteeRetreat.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+ 
+Below, we 1) summarize the motions that were passed at the meeting, 2) describe the five key areas over which the Steering Committee provides oversight, and 3) provide a timeline for implementing our goals.
+
+## Summary of motions passed at retreat
+
+- The Steering Committee has created a board-designated operating reserve ($80,000, equivalent to one quarter’s worth of budget) to provide an assurance of financial solvency
+- The Steering Committee empowers the trainer group to move forward with open instructor training events to non-member affiliated individuals 
+- The Steering Committee empowers staff to perform assigned tasks, with the Steering Committee retaining oversight in the key areas of community, instructor training, curriculum, finances, and staff
```
Comment (ErinBecker) [inline _posts/2017/06/2017-06-05-SteeringCommitteeRetreat.md]: Might I suggest "empowers staff to perform tasks in their areas of focus" . . . "and hiring staff"

File: _posts/2017/06/2017-06-05-SteeringCommitteeRetreat.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+#### Community
+We value the involvement of everyone in our community - learners, instructors, hosts, developers, maintainers, committee members, staff, partners, advocates, trainers, organizers, sponsors, advisors, and helpers. We are committed to creating a friendly and respectful place for learning, teaching and contributing. Will will continue to support the Code of Conduct, which is at the heart of our community. Our mission is to continue growing and supporting a  diverse and inclusive community. To that end, we have empowered a new Director of Community Engagement to accomplish this mission.
+ 
+#### Instructor training	
+We are committed to creating a community of practice around instruction. As we train new instructors and instructor trainers, we must also continue to support their professional development through a community of practice. We have empowered the Director of Instructor Training to expand the instructor training program to accommodate the needs of our community. Our strategic plan is identify new instructor trainees and provide training opportunities while simultaneously work to grow our capacity to offer mentoring and other support to those new instructors.
```
Comment (ErinBecker) [inline _posts/2017/06/2017-06-05-SteeringCommitteeRetreat.md]: A few typos in strategic plan:

"is to identify"
"simultaneously working"

File: _posts/2017/06/2017-06-05-SteeringCommitteeRetreat.md (hunk 1 above)
Comment (raynamharris) [reply _posts/2017/06/2017-06-05-SteeringCommitteeRetreat.md]: clarified

File: _posts/2017/06/2017-06-05-SteeringCommitteeRetreat.md (hunk 2 above)
Comment (raynamharris) [reply _posts/2017/06/2017-06-05-SteeringCommitteeRetreat.md]: fixed

**Comment by ChristinaLK** [comment]
Good summary.  +1 to post.

**Comment by raynamharris** [comment]
Okay, let's go ahead and publish. Okay to post @jduckles

**Pull request merged**